apps/portal/app/routes/app+/list+/$id+/index.tsx

```diff
@@ -27,112 +37,288 @@ import { ListTabIdentityDisplay } from '@components/lists/list-tab-identity-disp
 import RemixLink from '@components/remix-link'
 import SaveListModal from '@components/save-list/save-list-modal'
 import { DataHeaderSkeleton, PaginatedListSkeleton } from '@components/skeleton'
-import { useLiveLoader } from '@lib/hooks/useLiveLoader'
-import { getListClaims } from '@lib/services/lists'
 import { addIdentitiesListModalAtom, saveListModalAtom } from '@lib/state/store'
+import { getSpecialPredicate } from '@lib/utils/app'
+import logger from '@lib/utils/logger'
 import {
-  getAtomDescription,
-  getAtomImage,
-  getAtomIpfsLink,
-  getAtomLabel,
-  getAtomLink,
+  getAtomDescriptionGQL,
+  getAtomImageGQL,
+  getAtomIpfsLinkGQL,
+  getAtomLabelGQL,
+  getAtomLinkGQL,
+  identityToAtom,
   invariant,
 } from '@lib/utils/misc'
 import { defer, LoaderFunctionArgs } from '@remix-run/node'
 import {
-  Await,
+  useLoaderData,
   useNavigation,
   useRouteLoaderData,
   useSearchParams,
 } from '@remix-run/react'
-import { fetchWrapper } from '@server/api'
 import { requireUserWallet } from '@server/auth'
+import { QueryClient } from '@tanstack/react-query'
 import {
+  CURRENT_ENV,
   MULTIVAULT_CONTRACT_ADDRESS,
   NO_CLAIM_ERROR,
   NO_PARAM_ID_ERROR,
   NO_WALLET_ERROR,
 } from 'app/consts'
-import { VaultDetailsType } from 'app/types'
 import { useAtom, useSetAtom } from 'jotai'
 
 export async function loader({ request, params }: LoaderFunctionArgs) {
   const id = params.id
   invariant(id, NO_PARAM_ID_ERROR)
 
+  const [predicateId, objectId] = id.split('-')
```

Comment:
Good idea! When we fully migrate we can doublecheck that this ID is what we want to use but I think that using this composite makes a lot of sense